portfolio/src/main/webapp/css/custom.css

```diff
@@ -355,3 +355,9 @@ body {
 .dropdown:hover .dropbtn {
   background-color: #6ab5cc;
 }
+
+.rounded-circle {
+  border-radius: 50% !important;
+  height: 100%;
+  width: 100%;
```

Comment:
In the image with the timeline, the images still seem stretched. Maybe using object-fit: cover will fill the circle without stretching the image.

Reply:
object-fit: cover does not fill the circles appropriately. I will try finding an alternative.